src/common/components/header/index.js

```diff
@@ -9,13 +9,20 @@ export default class Header extends Component {
       <div className={ styles.header }>
         <nav className={ styles.container }>
           <Link className={ styles.logo } to="/">
-            build.snapcraft.io
+            snapcraft
           </Link>
           <div className={ styles.sideNav }>
-            <a href="http://snapcraft.io" className={ styles.link }>snapcraft.io</a>
+            { this.props.authenticated
+              ? <a href="/auth/logout" className={ styles.link }>Logout</a>
+              : <a href="/auth/authenticate" className={ styles.link }>Login</a>
```

Comment:
Should be "Log out" and "Log in" (design might want to make this "Sign out" etc. instead, but let's make it grammatically-correct at least).